AnnaDovga/HW6/1.py

```diff
@@ -0,0 +1,3 @@
+print('Number division 2: ', *[i for i in range(2, 11, 2)])
```

Comment:
good job
look at this version of code
even = []
odd = []
other = []

for number in range(1, 11):
    if number >= 2 and number % 2 == 0:
        even.append(number)
    elif number >= 3 and number % 3 == 0:
        odd.append(number)
    else:
        other.append(number)